client/components/date-range/index.js

```diff
@@ -279,8 +278,8 @@ export class DateRange extends Component {
 	 * Converts moment dates to a DateRange
 	 * as required by Day Picker DateUtils
 	 *
-	 * @param  {MomentJSDate} startDate the start date for the range
-	 * @param  {MomentJSDate} endDate   the end date for the range
+	 * @param  {Date} startDate the start date for the range
+	 * @param  {Date} endDate   the end date for the range
```

Comment:
If these are meant to be moment date instances then we should keep that accurate:

```suggestion
	 * @param  {import('moment').Moment} startDate the start date for the range
	 * @param  {import('moment').Moment} endDate   the end date for the range
```

Reply:
Sounds fine to me 👍